commit 4addbbb8b64065d09a154007a3b6a00add8fd6d0
Author: Isaac Davis <isaac.davis@joyent.com>
Date:   2019-05-01T22:52:56+00:00 (5 months ago)
    
    TOOLS-2251 Makefile.go_prebuilt.targ references local copy of download_go instead of eng.git path

diff --git a/tools/mk/Makefile.go_prebuilt.targ b/tools/mk/Makefile.go_prebuilt.targ
index 43c0757..0632da6 100644
--- a/tools/mk/Makefile.go_prebuilt.targ
+++ b/tools/mk/Makefile.go_prebuilt.targ
@@ -51,7 +51,7 @@ endif
 $(GO_TARBALL):
 	rm -f $@
 	mkdir -p $(@D)
-	tools/download_go $(GO_PREBUILT_VERSION) solaris amd64 $(CACHE_DIR)
+	./deps/eng/tools/download_go $(GO_PREBUILT_VERSION) solaris amd64 $(CACHE_DIR)
 
 #
 # Extract the Go toolchain.  This stamp includes the version number of the
